playbooks/roles/aws_logs_agent/tasks/main.yml

```diff
@@ -0,0 +1,19 @@
+---
+
+- name: Download the AWS Logs Agent python script
+  get_url:
+    url: "{{ AWS_LOGS_AGENT_SCRIPT_URL }}"
+    dest: /tmp/aws-logs-agent.py
+    mode: "1777"
+  tags:
+    - install
+    - install:base
```

Comment:
Are these tags are correct?

Reply:
Changed it to "install:code"

Reply:
@DubeySandeep, sorry, I should have been more clear here. These tags are needed to run some specific set of Ansible tasks. What is the goal of tagging this task?

Reply:
I've removed it as it's not needed!